features/steps/utils/vs_none.py

```diff
@@ -0,0 +1,56 @@
+"""
+
+In case of validating entity attribute lookup in the behave stack tree, 
+it woul dbe useful to differnetiate between a standard 'None' and a custom representation of 'None'
+that behaves similarly to the built-in `None` 
+but can be distinguished from it. The `VSNoneType` class follows the singleton pattern, ensuring 
+that only one instance of this custom `None` type exists.
+
+The custom `None` type is useful in scenarios where we need to differentiate between a standard 
+`None` and a custom representation of `None`. This is e.g. the case with validating entity attribute lookup in the behave stack tree. 
+
+Usage:
+    from custom_none import VSNone
+
+    print(VSNone)            # VSNone
+    print(VSNone == None)    # True
+    print(VSNone is None)    # False
+    print(bool(VSNone))      # False
+"""
+
+class VSNoneType:
```

Comment:
We should use a more descriptive and specific name for this custom type.  I recommend `NullAttribute` given that it refers to a blank attribute on an entity instance.

Reply:
That's indeed more descriptive https://github.com/buildingSMART/ifc-gherkin-rules/pull/197/commits/b635532c83191a5dc2d04f65af96648728b7f7e9